warp/aggregator/aggregator_test.go

```diff
@@ -23,7 +23,7 @@ func newValidator(t testing.TB, weight uint64) (*bls.SecretKey, *avalancheWarp.V
 	pk := bls.PublicFromSecretKey(sk)
 	return sk, &avalancheWarp.Validator{
 		PublicKey:      pk,
-		PublicKeyBytes: bls.PublicKeyToBytes(pk),
+		PublicKeyBytes: bls.PublicKeyToUncompressedBytes(pk),
```

Comment:
I think this would align with the prior `PublicKeyToBytes`
```suggestion
		PublicKeyBytes: bls.PublicKeyToCompressedBytes(pk),
```

Reply:
applied